src/plot_api/plot_api.js

```diff
@@ -302,7 +303,7 @@ Plotly.plot = function(gd, data, layout, config) {
                     .selectAll(query)
                     .remove();
 
-                fullLayout._infolayer.selectAll('g.rangeslider-container')
+                rangesliderContainers
```

Comment:
``.selectAll('g.rangeslider-container')`` is moved out of the loop, maybe that's where the save comes from.